packages/pangea-sdk/src/test/java/cloud/pangeacyber/pangea/audit/LogStreamRequest.java

```diff
@@ -0,0 +1,19 @@
+package cloud.pangeacyber.pangea.audit;
+
+import cloud.pangeacyber.pangea.BaseRequest;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.List;
+
+public final class LogStreamRequest extends BaseRequest {
+
+	@JsonProperty("logs")
```

Comment:
Do they have to be in a `logs` field? Will the backend will parse them as multiple events?
If they have to be inside `logs` field we could do a class that has at least this field.

Reply:
It's how one provider does it, yes, but future providers might not.